fission/common.go

```diff
@@ -32,7 +32,7 @@ func fatal(msg string) {
 func getClient(serverUrl string) *client.Client {
 
 	if len(serverUrl) == 0 {
-		fatal("Need --server or FISSION_URL set to your fission server.")
+		fatal("Need --server or FISSION_URL set to your fission server. Or try source <(fission get-env-variables)")
```

Comment:
We should eliminate this fatal error, and get the url from the API. Why make the user set a var that we know how to get automatically?